mkdocs.yml

```diff
@@ -5,9 +5,10 @@ repo_url: https://github.com/stevengj/mpb/
 docs_dir: 'doc/docs'
 site_dir: 'doc/site'
 
-theme_dir: 'doc/mpb-mkdocs-theme'
-theme: readthedocs
-
+theme:
+  name: readthedocs
+  custom_dir: mpb-mkdocs-theme
```

Comment:
not `doc/mpb-mkdocs-theme` ?

Reply:
Apparently not. From https://www.mkdocs.org/user-guide/styling-your-docs/:

To override any of the files contained in that theme, create a new directory next to your `docs_dir` (...) And then point your `mkdocs.yml` configuration file at the new directory:
```yaml
theme:
    name: mkdocs
    custom_dir: custom_theme
```